src/api/config/initializers/airbrake.rb

```diff
@@ -101,10 +101,10 @@ def ignore_by_backend_400_message?(message)
 def ignore_by_class?(notice)
   exceptions_to_ignore = ['ActiveRecord::RecordNotFound', 'ActionController::InvalidAuthenticityToken',
                           'CGI::Session::CookieStore::TamperedWithCookie', 'ActionController::UnknownAction',
-                          'AbstractController::ActionNotFound', 'ActionView::MissingTemplate',
-                          'Timeout::Error', 'Net::HTTPBadResponse',
+                          'AbstractController::ActionNotFound', 'ActionView::MissingTemplate', 'Bunny::TCPConnectionFailedForAllHosts',
```

Comment:
`Bunny::TCPConnectionFailedForAllHosts` happens when the host on which the RabbitMQ server is down. We don't manage that host, so we cannot do anything here.